i18n/th_TH.csv

```diff
@@ -7,4 +7,6 @@
 "Print", "พิมพ์"
 "Choose your installment terms","เลือกระยะเวลาการผ่อนชำระ"
 "Miminum order value is 3000 THB", "ยอดสั่งซื้อขั้นต่ำ 3000 บาท"
```

Comment:
Currency formatting. Needs to be consistent with other bits that were fixed

Reply:
that is unrelated to current pr, but I have it in mind.. everything will be changed to the magento currency formatting @jonrandy

Reply:
Given that the other currency formatting issues have been fixed in this PR, I would suggest that this is now directly related to this PR since not fixing it now results in a feature with inconsistent currency formatting. 

Actually, searching the repo for this string shows it is only in the translation file and nowhere else. Is it even still used?

Reply:
well.. when user make shopping and see that installment terms he see currency formatting in the same way as he sees currency formatting displayed on summary up on the right side of that installment option. it is ok.

that 3k thb not formatted value is visible when the user makes shopping and the minimum requirement is not met. Both will never be displayed from our plugin at the same time.

Also, I wish to remind that it is PR, it is not release, single PR solves the single issue, we will add stuff when we will be doing a release.

Reply:
_Actually, searching the repo for this string shows it is only in the translation file and nowhere else. Is it even still used?_

Some merge errors... i did merge from master ... this string exists in master branch in html, not sure why html was not merged here.

Reply:
@jonrandy I improved that message in #202